src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -113,8 +113,9 @@ public async Task<IEnumerable<PackageRestoreData>> GetPackagesInSolutionAsync(st
         /// <param name="solutionDirectory">Current solution directory</param>
         /// <param name="packageReferencesDict">Dictionary of package reference with project names</param>
         /// <returns>List of packages restore data with missing package details.</returns>
-	    public IEnumerable<PackageRestoreData> GetPackagesRestoreData(string solutionDirectory,
-        Dictionary<PackageReference, List<string>> packageReferencesDict)
+#pragma warning disable IDE0055
```

Comment:
I don't why IDE0055 (Fix formatting) analyzer reported a warning for the following method in the interface. Adding a suppression to GlobalSuppressions.cs file didn't resolve the warning. Hence added the suppression directly in the code.